cfme/tests/containers/test_ssl_providers.py

```diff
@@ -59,12 +60,20 @@ def test_add_provider_naming_conventions(provider, soft_assert):
             - Integer name
         * Assert that provider was added successfully with each of those
     """
+    token = provider.process_credential_yaml_key(
```

Comment:
How?, I'm passing the provider to the test so I can't share the token among the tests (it's possible that I just didn't understand you correctly).